dev/lint-python

```diff
@@ -207,7 +207,7 @@ PYTHON_SOURCE="$(find . -name "*.py")"
 compile_python_test "$PYTHON_SOURCE"
 pycodestyle_test "$PYTHON_SOURCE"
 flake8_test
-sphinx_test
+# sphinx_test
```

Comment:
Comment it by purpose? Just remove it?

Reply:
I was about to ask Shane to install Sphinx in Python 3, and recover it back later. I will fill a JIRA and comment together.